src/opencl/mod.rs

```diff
@@ -45,7 +45,8 @@ pub struct Device {
     compute_capability: Option<(u32, u32)>,
     pci_id: PciId,
     uuid: Option<DeviceUuid>,
-    device: opencl3::device::Device,
+    /// Handle to OpenCL device.
+    pub device: opencl3::device::Device,
```

Comment:
Why does it need to be public now?

Reply:
Appears to no longer be an issue with 1.56.1 toolchain